From 515e8ba48b12fc32dcd80ee8cf48ec833396c075 Mon Sep 17 00:00:00 2001
From: Sascha Peilicke <sascha@peilicke.de>
Date: Sat, 5 May 2018 11:16:24 +0200
Subject: [PATCH] Remove broken she-bang line

It's invalid as it doesn't start with '/', i.e. providing a path to the
bash binary nor is it using '/usr/bin/env bash'. Removing is best since
it's not needed for bash-completions.
---
 gibo-completion.bash | 2 --
 1 file changed, 2 deletions(-)

diff --git a/gibo-completion.bash b/gibo-completion.bash
index 16e8eba..a2a595f 100644
--- a/shell-completions/gibo-completion.bash
+++ b/shell-completions/gibo-completion.bash
@@ -1,5 +1,3 @@
-#!bash
-#
 # Bash completion for gibo
 #
 # INSTALLATION
-- 
2.17.0

